File: visaa_prep/season-game.java
import java.io.*;
import java.util.*;
public class Solution {
    public static void main(String[] args) {
        
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        if(1<=n && n<=12){
            if(3<=n && n<=5){
                System.out.println("Spring");
            }
            else if(6<=n && n<=8){
                System.out.println("Summer");
            }
            else if(9<=n && n<=11){
                System.out.println("Autumn");
            }
            else{
                System.out.println("Winter");
            }
        }
        else{
            System.out.println("Invalid");
        }
    }
}

File: visaa_prep/season-game.py
n=int(input())
if(1<=n<=12):
    if(n==4 or n==5 or n==3):
        print("Spring")
    elif(n==6 or n==7 or n==8):
        print("Summer")
    elif(n==9 or n==10 or n==11):
        print("Autumn")
    else:
        print("Winter")
  
else:
    print("Invalid")
